onnx/defs/schema.h

```diff
@@ -443,7 +458,7 @@ class OpSchema {
 class OpSchemaRegistry {
  public:
   // Update this when you make BC-breaking changes to the operator schema
-  constexpr static int version = 1;
+  constexpr static int version = 2;
```

Comment:
I think we need to have two global version numbers here, one for onnx, one for onnx-ml (we'll have two domains in onnx source tree, right?)

Let's also add comment to clarify a little bit more that this is the latest and biggest SinceVersion of an operator set and this "version" needs to be bumped accordingly when BC-breaking changes made on an operator set.

Reply:
A question: the extensibility scheme supports arbitrary domains; is the plan that this repository will provide special support for "onnx" and "onnx-ml"? Just trying to understand if it is okay to assume we have only these two domains in the implementation of the OpSchemaRegistry, or if we should design a more general mechanism.

Reply:
I didn't touch the domain mechanism because I knew you were going to be looking at it @linkerzhang :)

There are a bunch of ways to skin this cat, and I think the design should be related to how we do domains. IIRC, the plan for domains was to add a `Domain` field to OpSchema. In that case, it would probably make more sense for us to have map from domains to version numbers, to support however many version numbers we need to track.